ocaml/xapi/xapi_psr.mli

```diff
@@ -0,0 +1,60 @@
+(** psr = pool secret rotation
+  * ps = pool secret = ptoken *)
+
+type 'a r =
+  | Success
+  | Failed_during_accept_new_pool_secret of 'a
+  | Failed_during_send_new_pool_secret of 'a
+  | Failed_during_cleanup_slave of 'a
+  (* very unlikely - user needs to cleanup manually *)
```

Comment:
We can't tell users to delete files in dom0. We should just keep trying, for example on startup.

Reply:
:heart: that will make the code simpler